### PR TITLE
Add bpm-control and clean up code

### PR DESCRIPTION
**Pull request by FaGru**

### CODE REVIEW

File: src/components/SequencerSettings.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+  isSettingsVisible,
+  currentBpm,
+  setCurrentBpm,
+  setIsSettingsVisible,
+}) {
+  return (
+    <>
+      {isSettingsVisible ? (
+        <SettingsContainer>
+          <CloseButton aria-label='close' type='button' onClick={() => setIsSettingsVisible(!isSettingsVisible)}>
```
Comment (mwarnck) [inline src/components/SequencerSettings.js:14]: Wouldn´t it be better to alwas set isSettingsVisable to false with the close button?

File: src/components/VolumeControl.js [hunk 2]
```diff
@@ -11,7 +13,12 @@ export default function VolumeControl({
     <>
       {isControlsVisible ? (
         <ControlsContainer>
+          <Heading>
           <H2>Volume</H2>
+          <CloseButton type="button" aria-label='close' onClick={() => setIsControlsVisible(!isControlsVisible)}>
```
Comment (mwarnck) [inline src/components/VolumeControl.js:18]: Here the same

File: src/pages/SequencerPage.js [hunk 3]
```diff
@@ -23,6 +27,15 @@ export default function SequencerPage({ allPads }) {
   const [isSequencePlaying, setIsSequencePlaying] = useState('stopped');
   const [currentTimeStemp, setCurrentTimeStemp] = useState('');
 
+  // var AudioContext = window.AudioContext || window.webkitAudioContext;
+
+  // // var audioCtx = new AudioContext({
+  // //   latencyHint: 'interactive',
+  // //   sampleRate: 44100,
+  // // });
```
Comment (mwarnck) [inline src/pages/SequencerPage.js:35]: is this still needed?

Comment (FaGru) [reply src/pages/SequencerPage.js:35]: not yet, maybe later...

File: src/components/SequencerSettings.js (hunk 1 above)
Comment (FaGru) [reply src/components/SequencerSettings.js:14]: i think it doesn't matter, because i can only click the close button if the is state is true...

**Pull request merged**